contracts/test/behaviour/harvester.js

```diff
@@ -0,0 +1,241 @@
+const { expect } = require("chai");
+const { parseUnits } = require("ethers").utils;
+
+const { usdtUnits } = require("../helpers");
+
+/**
+ *
+ * @param {*} context a function that returns a fixture with the additional properties:
+ * - strategy: the strategy to test
+ * - rewards: array of objects with asset and expected properties
+ * @example
+    shouldBehaveLikeHarvester(() => ({
+        ...fixture,
+        strategy: fixture.convexStrategy,
+        rewards: [
+        { asset: fixture.crv, expected: parseUnits("2") },
+        { asset: fixture.cvx, expected: parseUnits("3") },
+        ],
+    }));
+ */
+const shouldBehaveLikeHarvester = (context) => {
+  describe("Harvester behaviour", () => {
+    it("Should allow the governor to call harvest for a specific strategy", async () => {
+      const { harvester, governor, strategy } = context();
+      // prettier-ignore
+      await harvester
+          .connect(governor)["harvest(address)"](strategy.address);
+    });
+
+    it("Should collect reward tokens using collect rewards on all strategies", async () => {
+      const { harvester, governor, rewards } = context();
+
+      // No rewards in the harvester before harvest
+      for (const reward of rewards) {
+        await expect(
```

Comment:
this `await` not needed. Same comment in other lines similar to this as well.  
Also, we have a `balanceOf` helper method. So, you can do something like `await expect(harvester).to.have.balanceOf(0, reward.asset)` as well.

Reply:
you are correct on both points. I'll change it to use the helper matcher for consistency. Thanks